desktop/src/main/java/bisq/desktop/components/paymentmethods/RevolutForm.java

```diff
@@ -84,6 +102,43 @@ public void addFormForAddAccount() {
         addCurrenciesGrid(true);
         addLimitations(false);
         addAccountNameTextFieldWithAutoFillToggleButton();
+
+        //set default country as selected
+        selectedCountry = CountryUtil.getDefaultCountry();
+        if (CountryUtil.getAllRevolutCountries().contains(selectedCountry)) {
+            countryComboBox.getSelectionModel().select(selectedCountry);
+        }
+    }
+
+    ComboBox<Country> addCountrySelection() {
+        HBox hBox = new HBox();
+
+        hBox.setSpacing(5);
+        ComboBox<Country> countryComboBox = new JFXComboBox<>();
+        hBox.getChildren().add(countryComboBox);
+
+        addTopLabelWithVBox(gridPane, ++gridRow, Res.get("payment.bank.country"), hBox, 0);
+
+        countryComboBox.setPromptText(Res.get("payment.select.bank.country"));
+        countryComboBox.setConverter(new StringConverter<>() {
+            @Override
+            public String toString(Country country) {
+                return country.name + " (" + country.code + ")";
+            }
+
+            @Override
+            public Country fromString(String s) {
+                return null;
+            }
+        });
+        return countryComboBox;
+    }
+
+    void setCountryComboBoxAction(ComboBox<Country> countryComboBox) {
+        countryComboBox.setOnAction(e -> {
+            selectedCountry = countryComboBox.getSelectionModel().getSelectedItem();
+            updateFromInputs();
```

Comment:
You need to reset the input field validation to make it work as expected. 
```
selectedCountry = countryComboBox.getSelectionModel().getSelectedItem();
accountIdInputTextField.resetValidation();
accountIdInputTextField.validate();
updateFromInputs();
```

Reply:
per your suggestion, added unfocus/refocus and this solved the problem. Pls check.